### ISSUE TITLE
Crashes

### ISSUE THREAD

**Issue by YukarinTheFloofy** [bruxo00/steam-auto-shutdown#9]
I wanted to use the program on my laptop, since it worked so flawlessly on my desktop.

It keeps on crashing as soon as I launch it though. I've tried each release and they all crash within a few seconds of opening. Any idea why?

**Comment by diogomartino** [comment]
I'm not sure. Does it give you any crash message? There are only two things I can think of:

- for some reason it crashes while trying to figure out which network card you are using
- you don't have permissions to access the Steam process (unlikely)

Try to run it as administrator. If that doesn't work, the only way to help you is if there is any crash message. I also might push an update to have better error handling, because at the moment there is basically none and that might help you figure out what's going on.

**Comment by YukarinTheFloofy** [comment]
Apologies for the late response. I was out camping. Strangely enough, when I came back, I redownloaded it to test it out, and it suddenly magically worked? 

My desktop does have a USB-connected WiFi adapter, if that plays any part. There was no crash message before, it seemed to work for a few seconds and then exit, no "Not Responding" messages or anything. However, for now there seems to be no problem. Appreciate the help regardless!

Edit: I just realised that I worded it wrong in the initial post. I meant to use it on my desktop, since it worked on my laptop well. Sorry lol
